pysat/_instrument.py

```diff
@@ -765,7 +765,7 @@ def __getitem__(self, key):
             # Slicing by date, inclusive.
             inst[datetime1:datetime2, 'name']
 
-            # Slicing by name and row/date
+            # Slicing by name and row/date (pandas only)
             inst[datetime1:datetime2, 'name1':'name2']
```

Comment:
Should we mock this behaviour for xarray?

Reply:
I thought about it, but implementation would be complex.  This also falls into an area that is "less precise" in that it assumes data variables will always be in the same order.  If something changes in a data source for a new version, it could break code, and I'm not inclined to encourage this usage.